homeassistant/components/tasmota_irhvac/const.py

```diff
@@ -0,0 +1,205 @@
+"""Provides the constants needed for component."""
+
+# All activity disabled / Device is off/standby
```

Comment:
Please use and import as many constants from [homeassistant/const.py](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/const.py) and [climate/const.py](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/const.py) as possible.

Reply:
Ok, I'load almost all constants from there ..

Reply:
Done!